package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@capacitor-community/fcm",
-  "version": "5.0.4-fiksu.1",
+  "version": "6.0.0",
```

Comment:
`-fiksu.1` was there so that we could release a new version of our changes without changing the version that this is based on. But this should work too.